### PR TITLE
[New] Managing the Kernel on a Linode

### PR DESCRIPTION
**Pull request by wildmanonline**
This guide replaces (and removes) the following guides:

- [How to Update your Linode's Existing Kernel](https://www.linode.com/docs/guides/update-kernel/)
- [How to Change your Linode's Kernel](https://www.linode.com/docs/guides/how-to-change-your-linodes-kernel/)

### CODE REVIEW

File: docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+---
+slug: managing-the-kernel-on-a-linode
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Use the Linode Manager to change which kernel your Linode boots.'
+keywords: ["kernel", "grub"]
+tags: ["linode platform","cloud manager"]
+license: '[CC BY-ND 4.0](http://creativecommons.org/licenses/by-nd/4.0/)'
+aliases: ['/tools-reference/custom-kernels-distros/run-a-distribution-supplied-kernel-with-kvm/','/platform/how-to-change-your-linodes-kernel/','/run-a-distribution-supplied-kernel-with-kvm/','/tools-reference/custom-kernels-distros/run-a-distribution-supplied-kernel/','/platform/update-kernel/', '/guides/how-to-change-your-linodes-kernel/']
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md:10]: I think an alias from the Update Your Linux Kernel guides is missing here: `/guides/update-kernel/`

File: docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+1.  Log in to the Linode through either [SSH](/docs/guides/connect-to-server-over-ssh/) or [Lish](/docs/guides/using-the-linode-shell-lish/).
+
+1.  Run one of the following commands to display the kernel version:
+
+    - `uname -r`: Outputs a single line with just the kernel version.
+    - `hostnamectl`: Outputs an organized list of system details, including the kernel version. Can only be used on distributions with [systemd](https://en.wikipedia.org/wiki/Systemd) (Debian, Ubuntu, CentOS, openSUSE, and more).
+    - `cat /proc/version`: Outputs a string that contains the kernel version and release details.
+
+The kernel version should be a string that's similar to `5.4.0-80-generic`. In this example, the kernel ends with `generic`, which is common in Ubuntu systems to designate that this is an upstream kernel. If the kernel ends with something similar to `linode123`, a Linode kernel is being used and the string exactly which Linode kernel.
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md:46]: ...a Linode kernel is being used and the string exactly *matches* which Linode kernel.

File: docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+1. Find your current *Configuration* and click the corresponding **Edit** link. This may be located within the **More Options Ellipses** dropdown menu.
+
+1.  Scroll to the *Boot Settings* section.
+
+1.  Review the **Kernel** dropdown menu selection.
+
+    - **Latest 64 bit**: Uses the latest 64-bit Linode kernel at the time the Linode boots/reboots. This was the default for most 64-bit distributions prior to August 2018.
+    - **Latest 32 bit**: Uses the latest 32-bit Linode kernel at the time the Linode boots/reboots. This was the default for most 32-bit distributions prior to August 2018.
+    - **Direct Disk**: Instead of a Linux Kernel, this uses the MBE (Master Boot Record) of the primary disk*.
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md:66]: I don't think you meant to have the `*` at the end of the sentence?

File: docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    - **GRUB 2**: Uses the upstream distribution-supplied kernel that's installed on the primary disk. In a custom kernel has been installed instead, that is used instead. **This is the most common option and has been the default for most new Linodes created after August 2018.**
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md:67]: *If* a custom kernel has been installed instead,...

File: docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+    - **GRUB (Legacy)**: Uses the upstream distribution-supplied kernel that's installed on the primary disk. This should only be used on older Linux distributions that have Grub (not Grub 2) installed, like CentOS 6.
+    - **Specific Linode Kernel**: Allows you to specify the exact Linode kernel to use. Since this kernel is maintained regardless of reboots or system updates, it may be preferred in some use cases.
+
+    *\*Primary disk: the disk assigned as the "Root Device" within the Linode's Configuration Profile.*
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md:71]: Is this line supposed to be here?

File: docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+## Updating the Kernel
+
+Provided a newer kernel is available, you should be able to either manually (or automatically) update the kernel. The exact steps vary depending on the type of kernel and, for upstream kernels, the Linux distribution you are using.
+
+### Updating the Linode Kernel
+
+Follow these steps if the Linode is using a Linode kernel:
+
+-   **Latest 64-bit or 32-bit kernel**: Rebooting the Linode automatically updates the kernel used within your system.
+-   **Specific kernel version** (ex: `5.12.2-x86_64-linode144`): To update your kernel, follow the instructions within the [Viewing and Modifying the Kernel](#viewing-and-modifying-the-kernel) section. When selecting the kernel in the Linode's Configuration Profile, chose your desired kernel version (or select `Latest 64 bit`), save the changes, and reboot your Linode.
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md:86]: The anchor is no longer working. It should be `#viewing-and-modifying-the-kernel-in-the-cloud-manager`

**Review by leslitagordita — CHANGES_REQUESTED** [review]

File: docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md (hunk 3 above)
Comment (wildmanonline) [reply docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md:66]: Actually, this `*` is referring to the note at the end of the list. Going to mark this as resolved but let me know if I should do this a different way.

File: docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md (hunk 5 above)
Comment (wildmanonline) [reply docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md:71]: Ah, yeah. I think defining primary disk down here, instead of the two places it exists in the above list, cleans things up a bit. The precise definition is a bit wordy so I thought I'd just put it once.

**Review by leslitagordita — APPROVED** [review]
